wp-content/themes/thinkery/style.css

```diff
@@ -671,7 +671,7 @@ th:last-child {
 
 .post-password-form input[type="submit"],
 .search-form input[type="submit"] {
-	margin-top: 10px;
+	margin-top: 0px;
```

Comment:
Great call, this aligns the "search" text within the input box with the menu items perfectly!